controllers/enrollmentkeys.go

```diff
@@ -113,12 +116,20 @@ func createEnrollmentKey(w http.ResponseWriter, r *http.Request) {
 		newTime = time.Unix(enrollmentKeyBody.Expiration, 0)
 	}
 
+	relayId, err := uuid.Parse(enrollmentKeyBody.Relay)
+	if err != nil {
```

Comment:
this will give an error if `enrollmentKeyBody.Relay` is empty string, skip parsing if `enrollmentKeyBody.Relay)` is not set